styles/style.css

```
@@ -0,0 +1,235 @@
+/* Reset */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

You are already including the reset file, but I'm not sure about the box-sizing snippet, so include it just in case.
```suggestion
html { box-sizing: border-box; }
*::before, *, *::after { box-sizing: inherit; }
```

---

It's done, thanks!